### PR TITLE
Add Nano Known issues and some notebook link

### PR DESCRIPTION
**Pull request by yangw1234**

### CODE REVIEW

File: docs/readthedocs/source/doc/Nano/Overview/known_issues.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Nano Known Issues
+
+## **PyTorch Issues**
+
+### **AttributeError: module 'distutils' has no attribute 'version'**
+
+This usually is because the latest setuptools does not compatible with PyTorch 1.9.
+
+You can downgrade setuptools to 58.0.4 to solve this problem.
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/Nano/Overview/known_issues.md:9]: can we specify the version in nano dependence?

File: docs/readthedocs/source/doc/Nano/Overview/known_issues.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+conda install setuptools==58.0.4
+```
+
+### **error while loading shared libraries: libunwind.so.8**
+
+You may see this error message when running `source bigdl-nano-init`
+```
+ Sed: error while loading shared libraries: libunwind.so.8: cannot open shared object file: No such file or directory.
+```
+You can teh following command to fix this issue.
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/Nano/Overview/known_issues.md:23]: use the following command

File: docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train.md [hunk 3]
```diff
@@ -40,13 +40,19 @@ trainer.fit(lightning_module, train_loader)
...
 
-Intel Extension for Pytorch (a.k.a. IPEX) extends PyTorch with optimizations for an extra performance boost on Intel hardware. BigDL-Nano integrates IPEX through the `Trainer`. Users can turn on IPEX by setting `use_ipex=True`.
+Intel Extension for Pytorch (a.k.a. IPEX) [link](https://github.com/intel/intel-extension-for-pytorch) extends PyTorch with optimizations for an extra performance boost on Intel hardware. BigDL-Nano integrates IPEX through the `Trainer`. Users can turn on IPEX by setting `use_ipex=True`.
 
 ```python
 from bigdl.nano.pytorch import Trainer
 trainer = Trainer(max_epoch=10, use_ipex=True)
 ```
 
+Note: BigDL-Nano does not install IPEX by default. You can install IPEX using the following command:
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train.md:50]: why not?

File: docs/readthedocs/source/doc/Nano/Overview/known_issues.md (hunk 1 above)
Comment (yangw1234) [reply docs/readthedocs/source/doc/Nano/Overview/known_issues.md:9]: When a user creates a conda environment and specifics the python version, setuptools will be installed by conda. 

If we list `setuptools` as a dependency in pip, there might be conflict with conda.

File: docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train.md (hunk 3 above)
Comment (jason-dai) [reply docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train.md:50]: how about ipex 1.11?

Comment (yangw1234) [reply docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train.md:50]: Yes, IPEX 1.11 can be added to `setup.py`

File: docs/readthedocs/source/doc/Nano/Overview/known_issues.md (hunk 2 above)
Comment (yangw1234) [reply docs/readthedocs/source/doc/Nano/Overview/known_issues.md]: fixed

File: docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train.md (hunk 3 above)
Comment (jason-dai) [reply docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train.md:50]: OK - then we need to understand what the performance will be without ipex (say, in ray tune) @MeouSker77

Comment (MeouSker77) [reply docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train.md:50]: > OK - then we need to understand what the performance will be without ipex (say, in ray tune) @MeouSker77

Ok, I'll do it.

**Pull request merged**